autode/calculation.py

```diff
@@ -91,23 +92,29 @@ def _get_energy(self, e=False, h=False, g=False, force=False):
             force (bool): Return the energy even if the calculation errored
 
         Returns:
-            (float): Energy in Hartrees, or None
+            (autode.values.Energy | None):
         """
         logger.info(f'Getting energy from {self.output.filename}')
+        kwargs = {'method': self.method, 'keywords': self.input.keywords}
 
-        if self.terminated_normally() or force:
+        if not self.terminated_normally() and not force:
+            logger.error('Calculation did not terminate normally. '
+                         'Energy = None')
+            return None
 
+        try:
             if h:
-                return self.method.get_enthalpy(self)
+                return Enthalpy(self.method.get_enthalpy(self), **kwargs)
```

Comment:
it does return an Enthalpy?!

Reply:
ah I see . Mostly because it’s cleaner to convert it there `get_enthalpy_value` ?

(this will be removed in the theremochemistry PR anyway, probably)

Reply:
it just feels like get_enthalpy should return an enthalpy aha, but it doesn't really matter.